src/sources/vector/mod.rs

```diff
@@ -88,6 +88,10 @@ impl SourceConfig for VectorConfig {
             VectorConfig::V2(v2) => v2.config.resources(),
         }
     }
+
+    fn can_acknowledge(&self) -> bool {
```

Comment:
Technically only `v2` can acknowledge. I think it's ok to leave as-is though as `v1` is deprecated and will be going away in a release or two.

Reply:
Yeah, I'm not sure how I could encode that easily, and didn't think it was a big deal given the deprecation.